### PR TITLE
limit post-length to 1500 symbols

### LINKED ISSUE #115: Limit post body to 1500 characters
Limit post body to 350 characters to avoid situations like this: https://yadi.sk/d/97tqF_pfg3R47

### PR DESCRIPTION
**Pull request by indeyets**
_This is a Work-in-progress pull-request_

The code expects that server responds with specific message and shows user-friendly version of it in an alert (should replace with some sane notification mechanism).

Will fix #115

### CODE REVIEW

File: public/js/app/controllers/TimelineGenericController.js [hunk 1]
```diff
@@ -177,6 +177,10 @@ define(["config",
                 }
               })
             }
+          }).catch(function(error) {
+            if (error.responseJSON.err == 'Maximum post-length is 1500 graphemes') {
+              alert("Posts longer than 1500 symbols are not allowed");
```
Comment (epicmonkey) [inline public/js/app/controllers/TimelineGenericController.js]: In the latest revision we have this:

```
this.displayMessage('Posts longer than 1500 symbols are not allowed')
```

where `this` is inherited from ApplicationController.

**Comment by indeyets** [comment]
@epicmonkey done

**Comment by epicmonkey** [comment]
@indeyets could you review this one more time, there are conflicts...

**Comment by indeyets** [comment]
@epicmonkey done. it is drastically simplier now (as you reimplemented large part of this code)

**Pull request merged**